demos/theme/_theme-shared.scss

```diff
@@ -61,10 +66,35 @@ body {
   background-color: $mdc-theme-background;
 }
 
+figure {
```

Comment:
This seems to be redundant of styles 10 lines above this. Remove this or remove those?

Reply:
Done